### ISSUE TITLE
Accessibility: player buttons are not labeled

### ISSUE THREAD

**Issue by beqabeqa473** [signalapp/Signal-Android#11518]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Play/pause and close buttons of a player are not labeled.

### Steps to reproduce
- Open a conversation, start playing voice message, try to read buttons of player at the top with talkback.

**Actual result:** Talkback doesn't read these buttons.
**Expected result:** Talkback should read description of these buttons.
### Device info

**Device:** Samsung Galaxy s21
**Android version:** 11.0.0
**Signal version:** Latest

**Comment by alex-signal** [comment]
Thanks for the report! I've added content descriptions to all the clickable fields of this view, and it'll go out in 5.20 (at which point this issue *should* autoclose)
